src/resources/resources.css

```diff
@@ -18,7 +18,8 @@
 	margin: auto;
 	margin-top: 6rem;
 	width: 80%;
```

Comment:
add max-width  at least 1200px will prevent stretching components when browser width is too wide. Keep 80% width.